app/request/utils.py

```diff
@@ -278,6 +306,18 @@ def _move_validated_upload(request_id, tmp_path):
         get_upload_key(request_id, valid_name),
         upload_status.READY)
 
+    # Store File Object
+    size = os.path.getsize(os.path.join(current_app.config['UPLOAD_DIRECTORY'] + request_id, valid_name))
```

Comment:
can replace `+` with a comma since `os.path.join` can take more than just 2 args to create a path